command-parser.js

```diff
@@ -7,7 +7,7 @@
  *
  * Individual commands are put in:
  *   commands.js - "core" commands that shouldn't be modified
- *   config/commands.js - other commands that can be safely modified
+ *   chat-plugins/info.js - other commands that can be safely modified
  *
  * The command API is (mostly) documented in config/commands.js
```

Comment:
Although putting it in `chat-plugins/` something.md isn't a horrible idea either. Someone think of a filename?